app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -20,8 +20,8 @@ def create
       if Rails.env.test?
         journal_entry = Quickbooks::Model::JournalEntry.new(id: 123)
       else
-        creator = ::Accounting::Quickbooks::TransactionCreator.new
-        journal_entry = creator.create_in_qb @transaction
+        reconciler = ::Accounting::Quickbooks::TransactionReconciler.new
```

Comment:
There is a bug in the `TransactionCreator` that will fail when trying to create objects in QBO. The `TransactionReconciler` still uses this class internally, so it still has the bug. I assume there is some relation to the `InterestCalculator` that needs to be sorted out here. I will need help from @hooverlunch or @linesarefuzzy for this.